packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -76,12 +80,14 @@ export const useRadio = (props: RadioProps): RadioState => {
     ...pressable.state,
     selected: selectedInfo.value === props.value,
     disabled: selectedInfo.disabled || disabled || false,
+    isBelow: labelPosition === 'below' ? true : false,
```

Comment:
I think you could just do `isBelow: labelPosition === 'below',`

Reply:
Fixed, thanks!